pkg/cmd/controller/controller_build.go

```diff
@@ -128,8 +133,9 @@ func NewCmdControllerBuild(commonOpts *opts.CommonOptions) *cobra.Command {
 	cmd.Flags().BoolVarP(&options.FailIfNoGitProvider, "fail-on-git-provider-error", "", false, "If enable then lets terminate quickly if we cannot create a git provider")
 
 	// optional git reporting flags
-	cmd.Flags().StringVarP(&options.TargetURLTemplate, "target-url-template", "", "", "The Go template for generating the target URL of pipeline logs/views if git reporting is enabled")
+	cmd.Flags().StringVarP(&options.TargetURLTemplate, "target-url-template", "", "", "The Go template for generating the target URL of pipeline logs/views if git reporting is enabled. If unspecified, a default will be used based on `--job-url-base`.")
 	cmd.Flags().BoolVarP(&options.GitReporting, "git-reporting", "", false, "If enabled then lets report pipeline success/failures to the git provider. Note this is purely tactical until we can do this natively inside tekton")
+	cmd.Flags().StringVarP(&options.JobURLBase, "job-url-base", "", "", "The base URL, such as 'https://dashboard.jenkins-x.live', for generating the target URL for pipeline logs if git reporting is enabled.")
```

Comment:
So is the plan to change how the build controller is configured as well? Will there a related pull request for the build controller Helm chart?

Reply:
This part is a short term fix specifically for being able to run Lighthouse on our own prod cluster to find any further problems, so we’re just going to configure this in the prod cluster. #6793 will need to be resolved before we go GA with Lighthouse, so that one would only need to configure either of these two flags if they’re not using the JX UI for some reason.